### PR TITLE
Zaverichintan patch 11688(not to be merged)

### PR DESCRIPTION
**Pull request by zaverichintan**
Kindly review the code, it needs more testing and wrapper function support (as at some places cookies are directly set without using wrapper function).

Signed-Off- zaverichintan5@gmail.com

### CODE REVIEW

**Comment by zaverichintan** [comment]
Kindly review the logic of the code, and approve it , will implement it across pma after approval

**Comment by ibennetch** [comment]
Thanks. I'm getting merge conflicts against master, could you fix them and push the changes back to your branch (there should be no need to open a new pull request, you should be able to simply push or force push them to the branch you already have).

**Comment by zaverichintan** [comment]
It's a proposal for wrapper function which will create a major change in entire pma code. 
Kindly do not Merge it, as it requires some Mentor ( contributor of pma) to help me out.

**Comment by ibennetch** [comment]
I understand. Normally when reviewing a pull request the first thing I do is pull it to my local machine for review but it makes sense why that wouldn't work.

**Comment by zaverichintan** [comment]
Kindly suggest me work around. This enhancement seems small but it creates error in entire pma. 
Some Mentor can help me make wrapper function and explain me the flow of cookies that would be great

File: libraries/Config.php [hunk 1]
```diff
@@ -1879,7 +1889,14 @@ public function setCookie($cookie, $value, $default = null,
             return $this->removeCookie($cookie);
         }
 
-        if (! isset($_COOKIE[$cookie]) || $_COOKIE[$cookie] !== $value) {
+    switch ($cookieArr) {
+    case 'pmaUser':$size = 2 ;break;
```
Comment (ibennetch) [inline libraries/Config.php]: I believe our coding standard requires `case

Comment (ibennetch) [inline libraries/Config.php]: I believe our coding standard requires `case` statements to be split out in to multiple lines; see http://pear.php.net/manual/en/standards.control.php

**Comment by ibennetch** [comment]
It would be great if one of the other developers could comment on this; there are a few design decisions that I'd like a second opinion about before commenting.

**Comment by zaverichintan** [comment]
New PR 
https://github.com/phpmyadmin/phpmyadmin/pull/11919

**Pull request closed without merging**